.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "duckdb-java"]
+	path = duckdb-java
+	url = https://github.com/hrl20/duckdb-java.git
```

should this be `https://github.com/duckdb/duckdb-java`?
```suggestion
	url = https://github.com/duckdb/duckdb-java
```

---

my fork includes some changes that have not been merged to duckdb-java. i can probably transfer the repo to under motherduckdb.

---

Oh is that this PR? https://github.com/duckdb/duckdb-java/pull/120
Maybe we can ping them again to review via Discord, it looks like it's been open for a month